mp/pyboard.py

```diff
@@ -36,13 +36,14 @@ def close(self):
         if self.con is not None:
             self.con.close()
 
-    def read_until(self, min_num_bytes, ending, timeout=10, data_consumer=None):
+    def read_until(self, min_num_bytes, ending, timeout=2, data_consumer=None, max_recv=sys.maxsize):
```

Comment:
Isn't 2 a very short timeout? It routinely takes longer than that for me.

Reply:
Because a timeout reconnection has been made previously, this is usually not such a long time without limiting the size of the received data.

Reply:
Ah okay, great.